src/autosuggest/plain-list.tsx

```diff
@@ -93,6 +93,7 @@ const PlainList = ({
             data-mouse-target={index}
             enteredTextLabel={enteredTextLabel}
             screenReaderContent={screenReaderContent}
+            isKeyboard={!usingMouse.current}
```

Comment:
Changes to the ref don't trigger re-renders. If you need to trigger re-renders -- use state. If not (if the value is only used in handlers) -- you need to pass the ref itself, otherwise the value of isKeyboard won't be up to date.